en/developer_guide/bundle_configuration.rst

```diff
@@ -66,6 +66,58 @@ This bundle will be used as a base for all the applications api, you will need t
         http_exception_controller:  'OpenOrchestra\BaseApiBundle\Controller\ExceptionController::showAction'
         token_expiration_time:  +1month
 
+ApiBundle
+~~~~~~~~~
+
+This bundle will be used in the applications api. You can configure :
+
+* The ``Facade`` used by each ``Transformer``
```

Comment:
It's weird to present a facade as a transformer dependency, because in fact the transformer is the 'slave' of the facade. We don't need a facade because we use a transformer, but we need a transformer because we use facades :)